getting-started/installation-apt-debian.md

```diff
@@ -61,6 +61,10 @@ a `postgres` superuser (used in the rest of the docs):
 sudo service postgresql restart
 ```
 
+>:TIP: Our standard binary releases are by default Timescale-Licensed. This means that you can 
+use all of our free Community capabilities and seamlessly activate Enterprise capabilities. To build 
+an Apache-Only version, pass `-DAPACHE_ONLY=1` to `bootstrap` when building from source.
```

Comment:
an "Apache-Only version" is not very clear. Maybe `To build an version of this software that contains source code that is only licensed under Apache License Version 2.0 ....`  Also maybe a link to our blog post or something?

Also maybe ` Our standard binary releases are by default Timescale-Licensed` -> `Our standard binary releases contains code that is licensed under both  Apache License Version 2.0  and the Timescale License`

Reply:
Resolved, and also updated another place where we use the same language